st2common/st2common/persistence/base.py

```diff
@@ -357,12 +361,19 @@ def _dispatch_trigger(cls, operation, trigger, payload):
 
 class ContentPackResource(Access):
     @classmethod
-    def get_by_ref(cls, ref):
+    def get_by_ref(cls, ref, only_fields: Optional[List[str]] = None):
+        """
+        :param: only_field: Optional lists if fields to retrieve. If not specified, it defaults to
```

Comment:
```suggestion
        :param: only_fields: Optional list of fields to retrieve. If not specified, it defaults to
```

PS it is so refreshing to see some type hints in the StackStorm code.